content/en/data_jobs/databricks.md

```diff
@@ -19,23 +19,21 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
 
 1. [Configure the Datadog-Databricks integration](#configure-the-datadog-databricks-integration) with your Databricks API token.
 1. [Install the Datadog Agent](#install-the-datadog-agent-on-your-databricks-clusters) on your Databricks cluster(s).
-1. [Add your Datadog API key](#add-your-datadog-api-key-in-databricks) in Databricks.
 
-### Configure the Datadog-Databricks integration 
+### Configure the Datadog-Databricks integration
 
 1. In your Databricks workspace, go to **Settings** > **Developer**. Next to **Access tokens**, click **Manage**.
 1. Click **Generate new token**, enter a comment, and click **Generate**. Take note of your token.
 1. In Datadog, open the Databricks integration tile.
-1. On the **Configure** tab, click **Add New**. 
+1. On the **Configure** tab, click **Add New**.
 1. Enter a workspace name, your Databricks workspace URL, and the Databricks token you generated.
    {{< img src="data_jobs/databricks/configure-token.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, and API token." style="width:100%;" >}}
 
 
 ### Install the Datadog Agent on your Databricks cluster(s)
 
-Use one of the following init scripts:
+Use the following init script:
 - [Install the Datadog Agent][8]
```

Comment:
```suggestion
The following init script will install the Datadog Agent on the cluster:
- [`datadog_databricks_job_monitoring_init.sh`][8]
```

When I read this I thought it linked to a docs page. Maybe adding the script filename will make it more obvious this is actually the script to copy?

Reply:
Agree this is clearer, updated it